showcase/tests/integration/components/hds/time/index-test.js

```diff
@@ -0,0 +1,282 @@
+    await render(hbs`
+      <Hds::Time @date={{this.friendlyLocalDate}} @display="friendly-local" @isOpen={{true}} />
+    `);
+    assert.dom('.hds-time').hasText('Sep 5, 2018, 2:07:32 PM');
+    // test tooltip content
+    assert.dom('.tippy-content').hasText(this.friendlyLocalDate.toISOString());
+  });
+
+  // friendly local using ISO date string
+  test('it should render the correct string in friendly local format when passing an ISO date string', async function (assert) {
+    await render(hbs`
+      <Hds::Time @date="05 September 2018 14:07:32" @display="friendly-local" @isOpen={{true}} />
+    `);
+    assert.dom('.hds-time').hasText('Sep 5, 2018, 2:07:32 PM');
+    // test tooltip content
+    assert
+      .dom('.tippy-content')
+      .hasText(new Date('05 September 2018 14:07:32').toISOString());
+  });
+
+  // friendly only using Date object
+  test('it should render the correct string in friendly only format when passing a Date object', async function (assert) {
+    this.set('friendlyOnlyDate', new Date('05 September 2018 14:07:32'));
+
+    await render(hbs`
+      <Hds::Time @date={{this.friendlyOnlyDate}} @display="friendly-only" @isOpen={{true}} />
+    `);
+    assert.dom('.hds-time').hasText('Sep 5, 2018');
+    // test tooltip content
+    assert.dom('.tippy-content').hasText(this.friendlyOnlyDate.toISOString());
+  });
+
+  // friendly only using ISO date string
+  test('it should render the correct string in friendly only format when passing an ISO date string', async function (assert) {
+    await render(hbs`
+      <Hds::Time @date="05 September 2018 14:07:32" @display="friendly-only" @isOpen={{true}} />
+    `);
+    assert.dom('.hds-time').hasText('Sep 5, 2018');
+    // test tooltip content
+    assert
+      .dom('.tippy-content')
+      .hasText(new Date('05 September 2018 14:07:32').toISOString());
+  });
+
+  // UTC display type
+
+  // UTC using Date object
+  test('it should render the correct string in UTC format when passing a Date object', async function (assert) {
+    this.set('utcDate', new Date('05 September 2018 14:07:32'));
+
+    await render(hbs`
+      <Hds::Time @date={{this.utcDate}} @display="utc" @isOpen={{true}} />
+    `);
+    assert.dom('.hds-time').hasText(this.utcDate.toISOString());
+    // test tooltip content
+    assert.dom('.tippy-content').hasText(this.utcDate.toISOString());
+  });
+
+  // UTC using ISO date string
+  test('it should render the correct string in UTC format when passing an ISO date string', async function (assert) {
+    await render(hbs`
+      <Hds::Time @date="05 September 2018 14:07:32" @display="utc" @isOpen={{true}} />
+    `);
+    let date = new Date('05 September 2018 14:07:32');
+    assert.dom('.hds-time').hasText(date.toISOString());
+    // test tooltip content
+    assert.dom('.tippy-content').hasText(date.toISOString());
+  });
+
+  // Relative display types
+
+  // TODO: Tests for times in the near future are failing (descriptions are off by 1)
```

Comment:
Having issues with the below tests currently.

Reply:
I don't think there's any way around this, as even a fraction of a second would cause the relative dates to go down from T to T-1. I suggest we update the `assert`s to use T-1.

Reply:
or, we add a few seconds to each test (resulting in T+n) and we `assert` against T

Reply:
I'm updating tests to use T - 1, thanks.